### ISSUE TITLE
[item] Living Ruby Pendant id=24110 (BB #357)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#357]
This issue was migrated from bitbucket.
**Original Reporter:** 
**Original Date:** 28.12.2010 21:53:15 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357

<hr>

The problem is in item "Living Ruby Pendant"
(http://www.wowhead.com/item=24110).
It has an aura that restores 6 hp per second for all your party members.
This thing should not depend on the bonus healing, but it depends.
I mean that it heals more than 6 hp, depending on healing bonus. 
For example if you have 2000 bonus healing, it heals 110 hp per second.

Sorry for my bad English =)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 24.06.2013 02:50:59 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4944512

<hr>

The below fix works. **_Changes made to SpellMgr.cpp**_

```
#!diff
             spellInfo->EffectTriggerSpell[0] = 33760;
             break;
        case 13897: // Fiery Weapon Enchant
+       case 31024: // Living ruby pendant
            spellInfo->AttributesEx3 |= SPELL_ATTR_EX3_NO_DONE_BONUS;
            break;
        case 38509: // Vashjs surge shock
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 24.06.2013 05:59:30 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4945382

<hr>

SPELL_ATTR_EX3_NO_DONE_BONUS - not defined!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 24.06.2013 06:03:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4945398

<hr>

Is this define right?
#define SPELL_ATTR_EX3_NO_DONE_BONUS              0x20000000            // 29

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 24.06.2013 09:07:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4946969

<hr>

Not working..........
heals too strong.....

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 25.06.2013 01:00:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4957396

<hr>

```
#!c++
#define SPELL_ATTR_EX3_NO_DONE_BONUS              0x20000000            // 29 Ignore caster spellpower and done damage mods?  client doesn't apply spellmods for those spells
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 25.06.2013 05:04:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4958513

<hr>

Not working.......... heals too strong.....

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 25.06.2013 05:05:04 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4958518

<hr>

+66 Heal per second on my priest

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 25.06.2013 05:06:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4958522

<hr>

Eh? Working for me, add me to Skype: Nicksauce

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 25.06.2013 05:17:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-4958593

<hr>

https://bitbucket.org/oregon/oregoncore/issue/108/spell-mind-flay-in-raid

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 09.01.2014 17:36:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/357/#comment-7656926

<hr>

Resolved issue #357 - Generic spells

Someone reported that the item called "Living Ruby Pendant" casts an aura that restores
health depending on ones healing bonus or spell power. Clearly this is not how this item
is designed to work, so after only a guess and a few minutes of testing it became obvious
that this problem is not only limited to this particular item. Other spells/auras of the
spell family 'generic' are having their spell effects scaled with the players healing bonus.

This issue was resolved by LordUsagi. Much needed thanks goes to him for making a lot of spells
and items like wands balanced and correct.

→ <<cset ac0577b6a549>>
